core/src/stored/reserve.cc

```diff
@@ -244,9 +244,9 @@ static bool UseDeviceCmd(JobControlRecord* jcr)
   if (rctx.append) { jcr->sd_impl->dcr->SetWillWrite(); }
 
   if (!jcr->sd_impl->dcr) {
-    BareosSocket* dir = jcr->dir_bsock;
-    dir->fsend(T_("3939 Could not get dcr\n"));
-    Dmsg1(debuglevel, ">dird: %s", dir->msg);
+    BareosSocket* dir2 = jcr->dir_bsock;
```

Comment:
This looks like a c&p error to me.   `dir` already is `jcr->dir_bsock` and i dont see where `jcr->dir_bsock` would get changed in this function that we would need to reread it.